### PR TITLE
new handwired keyboard: Jotlily60

### PR DESCRIPTION
**Pull request by jotix**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/jotlily60/info.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        "pid": "0x2332",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/info.json:17]: It's generally preferred that mouse key functionality is enabled. Command and console generally aren't needed by default.
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true
```

File: keyboards/handwired/jotlily60/info.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    },
+    "tapping": {
+        "term": 175
+    },
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/info.json:23]:
```suggestion
    "development_board": "promicro",
```

File: keyboards/handwired/jotlily60/info.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "D0", "D4", "C6", "D7", "E6", "B4", "B5" ],
+        "rows": [ "F7", "B1", "B3", "B2", "B6" ],
+    },
+    "dynamic_keymap": {
+	"layer_count": 4
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/info.json:30]:
```suggestion
        "layer_count": 4
```

File: keyboards/handwired/jotlily60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,68 @@
+// This is the default layout for the handwired/jotanck keyboard
+
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/keymaps/default/keymap.c:9]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/handwired/jotlily60/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+#define _QWERTY 0
+#define _FN     1
+#define _ADJUST 2
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/keymaps/default/keymap.c:12]:
```suggestion
enum layers {
    _QWERTY,
    _FN,
    _ADJUST
};
```

File: keyboards/handwired/jotlily60/readme.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+# jotlily60
+
+![jotlily60](https://i.imgur.com/I68WGmJ.jpg)
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/readme.md:3]:
```suggestion
![jotlily60](https://i.imgur.com/I68WGmJh.jpg)
```

File: keyboards/handwired/jotlily60/rules.mk [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# build options
+
+DYNAMIC_MACRO_ENABLE = yes
```
Comment (waffle87) [inline keyboards/handwired/jotlily60/rules.mk:3]: Given that the dynamic macro functionality is not used anywhere..
```suggestion
# This file intentionally left blank
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

File: keyboards/handwired/jotlily60/info.json [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "D0", "D4", "C6", "D7", "E6", "B4", "B5" ],
+        "rows": [ "F7", "B1", "B3", "B2", "B6" ],
+    },
+    "dynamic_keymap": {
+        "layer_count": 4
+    }
```
Comment (drashna) [inline keyboards/handwired/jotlily60/info.json:30]: This is the default value. No need to specify it.
```suggestion
```

File: keyboards/handwired/jotlily60/info.json [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "tapping": {
+        "term": 175
+    },
```
Comment (drashna) [inline keyboards/handwired/jotlily60/info.json:21]: This is generally more of a user config option.
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/jotlily60/info.json (hunk 9 above)
Comment (jotix) [reply keyboards/handwired/jotlily60/info.json:21]: thank you for spoting that

File: keyboards/handwired/jotlily60/readme.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+# jotlily60
+
+![jotlily60](https://i.imgur.com/I68WGmJh.jpg)
+
+A split keyboard kit made by jotix, inspired in the Lily58 keyboard.
+
+Keyboard Maintainer: [jotix](https://github.com/jotix)
+Hardware Supported: Arduino Pro Micro
```
Comment (fauxpark) [inline keyboards/handwired/jotlily60/readme.md:8]:
```suggestion
* Keyboard Maintainer: [jotix](https://github.com/jotix)
* Hardware Supported: Arduino Pro Micro
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**